### ISSUE TITLE
Etcd metrics endpoints answers on the wrong IP

### ISSUE THREAD

**Issue by VinceMD** [kubernetes-incubator/kube-aws#923]
I run Kube-aws V 0.9.7 and try to hit the etcd metrics endpoint for Prometheus to scrape.
I run 3 etcd. I use the ENI method.
The first etcd node (node 0) answers to curl requests on :2379/metrics using the ENI IP.

```
curl -L -v  --cacert ./ca.pem --cert ./etcd-client.pem --key ./etcd-client-key.pem https://NODE0_ENI_IP:2379/metrics --insecure
```

However If I use the same method for node1 and node2 I obtain `Failed to connect to 10.11.14.114 port 2379: Connection refused`

For etcd1 and etcd2, the trick is to use the instance IP. In which case it works.

To summarise:
* etcd0 answers to metrics request on ENI IP (eth1)
* etcd1 and etcd2 answers to metric request on instance IP (eth0)

It would be better for my Prometheus to hit the ENI as they will remain the same even if a node goes down and is replaced.
At the moment, if I loose etcd1 or etcd2, I'll have to change my prometheus config to point to the new IP address.
